harness/propertyHelper.js

```diff
@@ -1,5 +1,6 @@
 
 function isConfigurable(obj, name) {
+    var desc = Object.getOwnPropertyDescriptor(obj, name);
```

Comment:
I'd be fine with such a helper! Though note that it only partially addressed the general footgun problem as I understand it. `verifyDescriptor(obj, prop, { writable: true, configurable: false, enumerable: true})` followed by anything else doing something with obj.prop will fail (or perhaps spuriously pass).